freetextresponse/freetextresponse.py

```diff
@@ -159,7 +158,7 @@ def workbench_scenarios():
             'This is the message students will see upon '
             'submitting their response'
         ),
-        default=_('Your submission has been received'),
```

Comment:
That was a weird issue, if we try to translate default='str' in the xblock settings variable it blows up on compiling.  See above error png.